org.eclipse.m2e.core.ui/src/org/eclipse/m2e/core/ui/internal/wizards/WarningComposite.java

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Sonatype, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *      Sonatype, Inc. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.m2e.core.ui.internal.wizards;
+
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.jface.dialogs.Dialog;
+import org.eclipse.jface.layout.GridDataFactory;
+import org.eclipse.jface.resource.JFaceResources;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.swt.widgets.Text;
+
+/**
+ * Composite which can be used to show an IStatus message & image
+ *
+ * @author Matthew Piggott
+ */
+public class WarningComposite extends Composite {
```

Comment:
This doesn't seem to be used. It would be good to remove it then.

Reply:
> This doesn't seem to be used. It would be good to remove it then.

Its included in `MavenPomSelectionComponent` to show warnings, its now been a couple years since I wrote it but I believe I was making an attempt to simplify the class and it seemed like a potentially re-usable piece.